.github/workflows/release.yaml

```diff
@@ -80,7 +80,7 @@ jobs:
       if: ${{ github.event.inputs.include_repo_version == 'true' }}
       run: |
         poetry version "$JRNL_VERSION"
-        echo __version__ = \"$JRNL_VERSION\" > jrnl/__version__.py
+        echo -e "# This file is managed automatically by the GitHub release flow\n\nimport sys\n\n__version__ = \"$JRNL_VERSION\"\n\n# this makes the version available at `jrnl.__version__` without requiring a\n# `__init__.py` file in the *jrnl* root directory\n\nsys.modules[\"jrnl.__version__\"] = __version__" > > jrnl/__version__.py
```

Comment:
Instead of putting this all on one line, can you make this a bit more readable with something like this?

```yaml
{
echo "one"
echo "two"
echo "three"
} > jrnl/__version__.py
```

Reply:
Also, watch out for the double angle brackets (`>>`). This concatenates to a file instead of overwriting it, so this currently would keep bloating up the file forever instead of updating it.

Reply:
I've adjusted this to the multiline format you recommend. I really don't have a way to test this bash part,so let me know if it needs updating...